sbnobj/Common/Calibration/TrackCaloSkimmerObj.h

```diff
@@ -136,6 +172,10 @@ namespace sbn {
     int   start_process; //!< Start G4 process of the particle. Values defned as enum in StandardRecord
     int   end_process; //!< End G4 process of the particle. Values defined as enum in StandardRecord
 
+    std::vector<TrueHit> truehits0; //!< List of True "hits" of this particle on Plane 0
+    std::vector<TrueHit> truehits1; //!< List of True "hits" of this particle on Plane 1
+    std::vector<TrueHit> truehits2; //!< List of True "hits" of this particle on Plane 2
```

Comment:
I assume (and believe) that this is a deliberate and informed choice and that `0` really means "plane 0" and not e.g. view U or "best plane" etc.
But I would like a confirmation, just to be sure.

Reply:
That is correct.